packages/async-queue/rollup.config.js

```diff
@@ -0,0 +1,3 @@
+import baseConfig from '../../scripts/rollup.config';
```

Comment:
Rollup runs ESM?

Reply:
Honestly I have no clue what they do. They also support .mjs but when I did that it starting nagging about not being able to find the basic config unless I specifically added .mjs to it as well. Yet as .js it still supports ESM imports... So yeah no idea what they do.